pagopa/pn-frontend
# PN-5836-changed email/mail with e-mail in all texts in PF/PG

## Short description
Changed all occurrences to "mail" or "email" appearing in texts with "e-mail". There are a lot of occurrences in the contacts ("recapiti") section, and also two in the banner appearing above the notification list if the logged user has no registered contacts.

## List of changes proposed in this pull request
- Changed all occurrences in literals inside i18n files `recapiti` and `notifiche`,  for both PF and PG.

## How to test
Enter the PF/PG app, play around with the contacts page, and also check the banner above the notification list. There is a catalog of occurrences of "mail/email" in a comment on the reference issue in JIRA.  
**Beware**  
To test some of the occurrences, you'll need a user with some contact addresses registered. I don't know how to do this in DEV, so I mocked `Contacts.api` like this (same for PF and PG):
```
  getDigitalAddresses: (): Promise<DigitalAddresses> =>
    apiClient.get<DigitalAddresses>(CONTACTS_LIST()).then((response) => ({
      legal: response.data.legal
        ? [...response.data.legal.filter((address) => address.codeValid !== false), {
          addressType: "LEGAL",
          channelType: "PEC" as any,
          recipientId: 'PF-37654561-446a-4c88-b328-6699a8322b33',
          senderId: 'default',
          value: 'toto@toto.pec',
          pecValid: true,
          codeValid: true,
        }]
        : [],
      courtesy: response.data.courtesy 
        ? [...response.data.courtesy, 
        // {
        //   addressType: "COURTESY",
        //   channelType: "EMAIL" as any,
        //   recipientId: 'PF-37654561-446a-4c88-b328-6699a8322b33',
        //   senderId: '4db741cf-17e1-4751-9b7b-7675ccca472b',
        //   value: 'toto@toto.it'
        // },
        {
          addressType: "COURTESY",
          channelType: "EMAIL" as any,
          recipientId: 'PF-37654561-446a-4c88-b328-6699a8322b33',
          senderId: 'default',
          value: 'titi@titi.it'
        }
      ] 
        : [],
    })),
```
